Tests/Common/TestUtils.swift

```diff
@@ -17,6 +17,7 @@
 @testable import OktaOidc
 
 struct TestUtils {
+    
```

Comment:
Why is the extra newline needed here?

Reply:
In general, we don't have a rule about the line after type declaration. I see that it distinguish from file to file. 
Should we introduce a new item in Style guide about this?

Reply:
I'm not sure; lets see if it becomes a problem.  Ultimately, formatting aids readability, and so there are scenarios where an extra space helps readability, while others it doesn't.